.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: Java CI
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+        java: [8]
```

Comment:
What's the point to have a matrix for `os` and `java` having only one single value ?

Reply:
Just left the matrix in, in case you would like to add other operating systems or jdk‘s to test against.